src/types/onyx/PolicyTag.ts

```diff
@@ -8,6 +8,7 @@ type PolicyTag = {
     /** "General Ledger code" that corresponds to this tag in an accounting system. Similar to an ID. */
     // eslint-disable-next-line @typescript-eslint/naming-convention
     'GL Code': string;
+    tags: PolicyTags;
```

Comment:
```suggestion

    tags: PolicyTags;
```

Please add a comment